src/app/medicine/medicine.cpp

```diff
@@ -19,7 +43,11 @@ std::map<std::string, std::string> Medicine::getInfo() const
     {
         comp += el + ", ";
     }
-    return {{{"expiration date"}, {std::to_string(expiration_date_)}},
+    auto expDateDay = std::to_string(static_cast<unsigned>(expiration_date_.day()));
+    auto expDateMonth = std::to_string(static_cast<unsigned>(expiration_date_.month()));
+    auto expDateYear = std::to_string(static_cast<int>(expiration_date_.year()));
+    auto expDate = expDateDay + "." + expDateMonth + "." + expDateYear;
```

Comment:
const auto

Reply:
I am not sure if it would work (especially considering format), but instead of creating string manualy we could try:
```
std::ostringstream oss;
oss << expiration_date;
std::string expiration_text = oss.str();
```

Reply:
Format with dot do not allow using stringstream with oss << expiration_date;